### PR TITLE
Fix RX model preview for different rates system

### PR DESCRIPTION
**Pull request by McGiverGim**
This a fast fix for the quad preview model in the RX tab. Now, with ACTUAL as default, it was slow, almost stopped, because it used the ACTUAL rates as BETAFLIGHT rates. This fixes the preview depending on the rates.

I think a better fix is to integrate this in the `RateCurve.js` file, but I didn't want to touch too much code in the RC phase that we are now, because the pid tuning tab will need to be adapted too to the changes.

### CODE REVIEW

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> FAIL
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> FAIL
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

File: src/js/tabs/receiver.js
Comment (haslinghuis) [inline src/js/tabs/receiver.js:882]:
```suggestion
            this.deadband,
            FC.RC_TUNING.pitch_rate_limit,
```

Comment (McGiverGim) [reply src/js/tabs/receiver.js]: Fixed! Thanks!

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by Asizon — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> PASS
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> FAIL
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Comment by ctzsnooze** [comment]
Now my model spins uncontrollably fast :-(

**Pull request merged**
